lib/active_interaction/backports.rb

```diff
@@ -17,5 +17,53 @@ def []=(name, value)
   class Errors
     # Required for Rails < 3.2.13.
     protected :initialize_dup
+
+    # Required for Rails < 5.
+    #
+    # Extracted from active_model-errors_details 1.2.0. Modified to add support
+    # for ActiveModel 3.2.0.
+    module Details
+      extend ActiveSupport::Concern
+
+      CALLBACKS_OPTIONS = ::ActiveModel::Errors::CALLBACKS_OPTIONS
+      private_constant :CALLBACKS_OPTIONS
+
+      included do
+        attr_reader :details
+
+        %w[initialize initialize_dup add clear delete].each do |method|
+          alias_method "#{method}_without_details", method
+          alias_method method, "#{method}_with_details"
```

Comment:
:+1: 

At some point along the way, it was not working for me. Now it is. I am not sure what changed.